packages/near-lake-primitives/tsdocs.json

```
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es2019",
+    "declaration": true,
+    "outDir": "./dist",
+    "allowUnreachableCode": false,
+    "allowUnusedLabels": false,
+    "exactOptionalPropertyTypes": true,
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noPropertyAccessFromIndexSignature": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "strictNullChecks": true,
+    "strictPropertyInitialization": true,
+    "strict": true
+  },
+  "typedocOptions": {
+    "entryPoints": ["src/types/core/*.ts", "src/types/*.ts"],
+    "exclude": ["src/types/index.ts", "src/types/core/index.ts"],
```

```suggestion
    "exclude": ["src/types/index.ts", "src/types/core/index.ts", "src/types/lakeContext.ts"],
```
There is nothing special about lakeContext.ts right now so lets remove it for the sake of clarity